docs/reinterpreting_effects.md

```diff
@@ -234,7 +234,7 @@ instance (MonadIO m, Algebra sig m) => Algebra (Http :+: sig) (MockHttpClient m)
     L (SendRequest req) -> do
       responder <- MockHttpClient ask
       (<$ ctx) <$> liftIO (responder req)
-    R other -> MockHttpClient (alg (runMockHttpClient . hdl) other ctx)
+    R other -> MockHttpClient (alg (runMockHttpClient . hdl) (R other) ctx)
```

Comment:
I usually use `inj` here - is there any preference to what we should do?

Reply:
I updated the doc to use `inj`. Is there any advantage of using it?

Reply:
Sorry, I wasn't trying to /suggest/ `inj`, just that I use it. I don't know which is better!

Reply:
I'm new to `fused-effects`, so I wouldn't know. But having to import `inj` from `Control.Effect.Sum` indicates, to me at least, that it's not meant to be used directly. Maybe @patrickt or  @robrix can weigh in.

Reply:
Generally I use `R`, but `inj` is fine too. As @shinzui notes, `inj` requires an extra import, so I’d probably use `R` in the docs.

Reply:
Cool, I'll probably switch to using `R` too then!